en/docs/adv_examples/yolov3.md

```diff
@@ -1,104 +1,104 @@
 ## YoloV3
 
-## 1.简介
+## 1. Introduction
 
-[YOLO](https://pjreddie.com/darknet/yolo/)系列的算法(经典的v1~v3)，是单阶段目标检测网络的开山鼻祖，YOLO—You only look once，表明其单阶段的特征，正是由于网络简单，单阶段的效率较快，使其区别于Faster-RCNN为代表的两阶段目标检测器，从一开始推出至今，便以速度快和较高的准确率而风靡目标检测领域，受到广泛使用和好评。
+[YOLO](https://pjreddie.com/darknet/yolo/) series of algorithms (v1~v3), is the first single-stage object detection network, YOLO — You Only Look Once indicates its single-stage feature. It is precisely because of the simple network and fast single-stage efficiency that it is different from the two-stage detector represented by Faster-RCNN. Since its introduction, it has become popular in the field of the target detection with its fast speed and high accuracy, and has been widely used and praised. 
```

Comment:
It is precisely because of the simple network and fast single-stage efficiency that it is different from the two-stage detector represented by Faster-RCNN. 
->
Because the network is simple and the single-stage efficiency is fast, it is distinguished from the two-stage target detector represented by fast RCNN.

Reply:
Since its introduction -> Since it was released